es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,53 @@
+import { createElement, noop } from '../lib';
+import { TodoList } from './todo-list';
+import { TodoForm } from './todo-form';
```

Comment:
After you update these files to use `export default`, you wont need the curly brackets `{ }` here